src/coverage-system/sectionfinder.ts

```diff
@@ -101,53 +93,54 @@ export class SectionFinder {
     }
 
     /**
-     * Takes paths and tries to make them absolute
-     * based on currently open workspaceFolders
-     * @param path potential partial path to be converted
+     * Calculates relativePath and workspaceFolder for given TextEditor instance
```

Comment:
very nice! this should hopefully work well in the multiple workspace scenario but did you check that? https://code.visualstudio.com/docs/editor/multi-root-workspaces

Reply:
I haven't test it - but I am relaying on `workspace.getWorkspaceFolder` to work as expected. According to [Adopting-Multi-Root-Workspace-APIs](https://github.com/Microsoft/vscode/wiki/Adopting-Multi-Root-Workspace-APIs) it should be OK.